src/main/java/nextstep/courses/domain/session/FreeSession.java

```diff
@@ -0,0 +1,16 @@
+package nextstep.courses.domain.session;
+
+import nextstep.payments.domain.Payment;
+
+public class FreeSession implements Enrollment {
+	Session session;
```

Comment:
```suggestion
	private Session session;
```
인스턴스 변수는 특별한 이유가 없는 한 private으로 구현